src/core/components/index.js

```diff
@@ -0,0 +1,18 @@
+exports.goOnline = require('./go-online')
```

Comment:
if it is inside an object, for every add/delete, we get the 'no dangling comma situation'